jsx/ato/edit_ato.jsx

```diff
@@ -4,7 +4,7 @@ import { ErrorFor, ShowError } from '../utils/validation_helper'
 import { showAlert, clearAlert } from 'utils/alert'
 import classNames from 'classnames'
 import { Field } from 'formik'
-import BooleanSelect from '../temperature/boolean_select'
+import BooleanSelect from '../ui_components/boolean_select'
```

Comment:
So migration of common components will be in /ui_components? sounds nice
May we add all common buttons in it? like add/delete/edit ?

Reply:
Delete and Edit were integrated into the collapsible list, which probably should be renamed.  I think it makes sense to consider making components from other common input patterns.  I also think the duration inputs are a candidate to be turned into a component and reused.

Reply:
I think an inventory of those components and refactor would really help us for further developments. Nice initiativ!